components/gauge-circular/pointers.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Pointers
+page_title: Circular Gauge - Pointers
+description: Circular Gauge for Blazor - Pointers.
+slug: circular-gauge-pointers
+tags: telerik,blazor,circular,gauge,pointers
+published: True
+position: 10
+---
+
+# Circular Gauge Pointers
+
+The pointers are the values that will be marked on the scale. You can customize them through the parameters they expose:
+
+* [LineCap](#linecap)
+
+* [PlaceholderColor](#placeholdercolor)
+
+* [Color](#color)
+
+* [Size](#size)
+
+## LineCap
+
+The `LineCap` parameter controls the shape of the scale ending and takes a member of the `CircularGaugePointerLineCap` enum:
+
+* `Round` - by default the shape of the scale ending would be round
+
+* `Butt` 
+
+>caption Change the shape of the scale. The result from the code snippet below.
+
+![Round shape](images/linecap-parameter.png)
+
+````CSHTML
+@* Use a flat shape for the end of the scale *@
+
+<TelerikCircularGauge>
+    <CircularGaugeScales>
+
+        <CircularGaugeScale>
+            <CircularGaugeScaleLabels Visible="true" />
+        </CircularGaugeScale>
+
+    </CircularGaugeScales>
+
+    <CircularGaugePointers>
+
+        <CircularGaugePointer Value="40" LineCap="@CircularGaugePointerLineCap.Butt">
```

Comment:
The pointer setting should be first in the markup.

Reply:
Done